Added HTML examples for tables and lists (#248)
carpentries-incubator/jekyll-pages-novice: _episodes/introduction.md

> In the `Hello world` example above 5 different tags are used (`html`, `head`, `title`, `body` and `p`) in their open `<>` and closed `</>` form.
> We see also the special `doctype` tag that indicates the format and version of the document, in this case, [HTML(5)][html5-wikipedia].
>

> > Other examples of frequently used tags are for: 

Reviewer: ```suggestion
> Many other tags exist to define: 
```

>
> Given the stylized text:
>
> <h1><em>Hello</em>, World!</h1>
>
> write the HTML that will produce the same result.
> **Hint** the big font is achieved by use of a heading.
>
> > ## Solution
> >
> > ~~~
> > <h1><em>Hello</em>, World!</h1>
> > ~~~
> > {: .language-html }
> {: .solution }
{: .challenge }      

Let's write a more complex HTML example using a table showing the "Hello, World!" text in different languages that renders like: 
![HTML table example](../fig/html-table.png){: width="600px"}

The HTML to produce such a table looks like this (you can copy+paste the snippet into the HTML file you created in the previous example):
~~~
<table>
    <tr><th>Language</th><th>Text</th></tr>

>     <tr><td>English </td><td>Hello, World!</td></tr>  

Reviewer: ```suggestion
    <tr><td>English</td><td>Hello, World!</td></tr>  
```

    <tr><td>French</td><td>Bonjour, le monde!</td></tr>
    <tr><td>Portuguese</td><td>Olá, Mundo!</td></tr>
    <tr><td>Serbian</td><td>Zdravo, svete!</td></tr>
</table>
~~~ 
{: .language-html }

> Each row in enclosed between `<tr>` and `</tr>` tags; within a row, `<th>` and `</th>` tags are used to contain table headings 

Reviewer: ```suggestion
Each row in enclosed between **t**able **r**ow `<tr>` and `</tr>` tags; within a row, `<th>` and `</th>` tags are used to contain **t**able **h**eadings. 
```

> (special table cells displayed in bold font) while regular table data cells are contained within `<td>` and `</td>` tags.

Reviewer: ```suggestion
(special table cells displayed in bold font) while regular **t**able **d**ata cells are contained within `<td>` and `</td>` tags.
```

A similar example written using HTML lists would look as follows:
![HTML list example](../fig/html-list.png){: width="600px"}
~~~
<ul>
    <li>English: Hello, World!</li>  
    <li>French: Bonjour, le monde!</li>
    <li>Portuguese: Olá, Mundo!</li>
    <li>Serbian: Zdravo, svete!</li>
</ul>
~~~ 
{: .language-html }

> Here, we used unordered list tags `<ul>` and `</ul>` to define a list with 4 items, each in turn wrapped in 

Reviewer: ```suggestion
Here, we used **u**nordered **l**ist tags `<ul>` and `</ul>` to define a list with 4 items, each in turn wrapped in 
```

> individual list item (`<li>` and `</li>`) tags. 

Reviewer: ```suggestion
individual **li**st item (`<li>` and `</li>`) tags. 
```
